### PR TITLE
Initial ARM Template

### PR DESCRIPTION
**Pull request by mwevill**

### CODE REVIEW

File: Azure/template.json [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+                        "databaseSkuName": {
+                            "value": "[parameters('databaseSkuName')]"
+                        },
+                        "databaseTier": {
+                            "value": "[parameters('databaseTier')]"
+                        }
+                    }
+                }
+            }
+    ],
```
Comment (adam230594) [inline Azure/template.json:247]: missing a curly bracket

File: Azure/template.json [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+                                "name": "LoggingRedisConnectionString",
+                                "value": "[parameters('loggingRedisConnectionString')]"
+                            },
+                            {
+                                "name": "ConfigurationStorageConnectionString",
+                                "value": "[parameters('configurationStorageConnectionString')]"
+                            },
+                            {
+                                "name": "AppName",
+                                "value": "das-assessor-functions"
```
Comment (adam230594) [inline Azure/template.json]: needs to be updated

File: Azure/template.json [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+                }
+            },
+            "dependsOn": [
+                "function-app-insights",
+                "worker-app-service-plan"
+            ]
+        },
+        {
+            "apiVersion": "2017-05-10",
+            "name": "sql-server-deploy",
```
Comment (adam230594) [inline Azure/template.json]: sql-database

File: Azure/template.json (hunk 1 above)
Comment (mwevill) [reply Azure/template.json:247]: Fixed

File: Azure/template.json (hunk 2 above)
Comment (mwevill) [reply Azure/template.json]: DOH!!!! Fixed

**Review by adam230594 — APPROVED** [review]

**Pull request merged**